integration/dbutils/migrator.spec.ts

```diff
@@ -12,6 +12,14 @@ var dbShape : Shapes.DBShape = {
   tables: [{
     tableName: 'users',
     indices: ['userName', 'email']
+  },
+  {
+    tableName: 'userData',
+    indices: []
+  },
+  {
+    tableName: 'activation',
```

Comment:
Oh, right! I forgot we decided we weren't going to send the user ID, just the activation code. So I see what you're getting at.

I'm okay with that then. So we will use the activation table for other things too, right? Such as for holding the code for a password reset? I know that's not something we need to worry about, but just wondering if this table is open to having more utility beyond the one time activation.